packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js

```diff
@@ -289,8 +290,27 @@ export class CategoryPage extends PureComponent {
             return null;
         }
 
+        /**
+         * Get the HEX value of the current primary color
+         * and transform it to the CSS Filter property
+         * supported value
+         */
+        const primaryBaseColor = getComputedStyle(document.documentElement)
+            .getPropertyValue('--primary-base-color').trim();
+        const cssFilter = hexToCSSFilter(primaryBaseColor);
+
+        /**
+         * Apply the value as the CSS variable value
+         * Since the filter string contains a semi-colon –
+         * we remove it
+         */
+        const { filter = '' } = cssFilter;
+        const style = {
+            '--primary-base-filter': filter.replace(/[;]/, '')
```

Comment:
You don't need a regexp to remove the trailing character.
You don't need a regexp to remove a single character.